src/ui/public/vislib/lib/types/point_series.js

```diff
@@ -8,7 +8,7 @@ export function VislibTypesPointSeries() {
     }) : null;
 
 
-    let interpolate = matchingSeriParams ? matchingSeriParams.interpolate : cfg.interpolate;
+    let interpolate = cfg.interpolate;
     // for backward compatibility when loading URLs or configs we need to check smoothLines
     if (cfg.smoothLines) interpolate = 'cardinal';
```

Comment:
not sure about the duplication of this here and in `vis_update`. is there a way we can consolidate this?

Reply:
the part in vis_update takes care of actually upgrading visualization config to the new format.
this part here makes sure vislib renders correctly with old config. While the code appears similar (and some conditions might be) its not really the same.